examples/nextjs-blog/README.md

```diff
@@ -0,0 +1,132 @@
+# MarkdownDB Next.js Blog Tutorial
+
+This tutorial guides you through creating a simple Next.js-based blog using MarkdownDB. MarkdownDB empowers you to treat markdown files as a database, simplifying content management and querying.
+
+## Step 1: Set Up a Next.js Project
+
+Begin by creating a Next.js project. If Next.js is not installed, execute the following command:
+
+```bash
+npx create-next-app my-blog
+cd my-blog
+```
+
+## Step 2: Create a Folder for Blog Posts
+
+Establish a folder to store your blog posts within your project. For instance:
+
+```bash
+mkdir content
+cd content
+```
+
+Inside the content folder, create three sample blog posts using markdown, such as:
+
+```bash
+- post-1.md
```

Comment:
I'd just have 2 posts ...

And i'd add some content here ... (we are going to at least need the front matter ...)